profile/tests/test_auth.py

```diff
@@ -0,0 +1,83 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+from ..models import User
+
+
+class UserAuthTests(APITestCase):
```

Comment:
Seeing that you are using some test values like "test@test.com" and "test123455" multiple times, maybe you could define them in a `setUpTestData` class method.